### PR TITLE
revert the env_vars removal for two exporters

### PR DESCRIPTION
**Pull request by anarcat**
We need the env_vars for the Debian package to work properly. In
Debian-land, the systemd `.service` file gets its parameters from the
EnvironmentFile, which we deploy through the `env_vars` here.

Yes, it's backwards, and no, it's not great because it doesn't
actually use the `options` stuff, but at least it works.

This is necessary to have those exporters be able to accept options
when using init_style=none and install_method=package.

This is a split off of #303 which should at least pass
tests (/probably/ because the `install_method=package` code path is
basically untested).

See also #32

### CODE REVIEW

**Comment by anarcat** [comment]
a cleaner approach for this is #530
